python/tskit/trees.py

```diff
@@ -1210,6 +1210,19 @@ def is_leaf(self, u):
         """
         return len(self.children(u)) == 0
 
+    def is_isolated(self, u):
+        """
+        Returns True if the specified node is isolated in this tree: that is
+        it has no parents and no children. Sample nodes that are isolated
+        and have no mutations above them are commonly used to represent
```

Comment:
I would delete the "commonly" here .

Reply:
Sure - I just wanted to avoid implying that they are *always* used to represent missing data, given the recent discussion about `ISOLATED_AS_MISSING`. Have you got an idea for another adjective, or do you think simply "are used to represent missing data" is OK?

Reply:
Just delete it I'd say - people can read the linked documentation for details.

Reply:
OK. adjective deleted.